config/smartcosmos-ext-interactions.yml

```diff
@@ -0,0 +1,16 @@
+server:
+  port: 45556
```

Comment:
shouldn't the port be set like this `port: ${smartcosmos.service.ext-interactions.port}`